chargeOclock/Source/Other/NetworkAPI.swift

```diff
@@ -39,6 +45,10 @@ class NetworkAPI {
 	private init() {
 	}
 
+	fileprivate func performPostRequest(to uri: String, header: [String : String]? = nil, body : [String : String]? = nil, completion: @escaping NetworkCompletion) {
```

Comment:
header => headers

although I don't think we need the headers parameter for now. the type for `body` is wrong if it's meant to be JSON.

please don't put spaces before the colon, neither for labels nor for dictionaries, ...